doc/source/user_guide/post.rst

```diff
@@ -23,6 +23,83 @@ streamed back to the client for review or visualization.
    via a Python client.
 
 
+
+Enriched Command Output
+~~~~~~~~~~~~~~~~~~~~~~~
+All :class:`Mapdl <ansys.mapdl.core.mapdl._MapdlCore>` commands output a string object which can be parsed to 
+obtain specific data from it.
```

Comment:
Get in the habit of wrapping to 80 characters.

```suggestion
All :class:`Mapdl <ansys.mapdl.core.mapdl._MapdlCore>` commands output a string
object which can be parsed to obtain specific data from it.
```

Do this for all future rst.

Reply:
Sure.